packages/api/src/shared/config.ts

```diff
@@ -16,7 +16,7 @@ export const getEnvVarOrFail = (varName: string): string => coreGetEnvVarOrFail(
 export class Config {
   // env config
   static readonly PROD_ENV = "production";
-  static readonly DEV_ENV = "dev";
+  static readonly DEV_ENV = "development";
```

Comment:
The dev docker ENV_TYPE is set to development. Was getting errors locally because Config.isDev() rendered false